### ISSUE TITLE
Error retrieving data when opening landing page

### ISSUE THREAD

**Issue by davroth** [patrickomni/omnimobileserver#244]
1. Browse to https://nettelpong.azurewebsites.net/sv, chrome 43, windows 8.1, login as dave
2. Select Reisner and observe Landing Page

Expected - landing page displays w/o error

Actual - landing page displays red error popup msg below 
"There was an error getting some of the data to be graphed. Any data shown may not be complete. Please contact Omni about this error.undefined"

This also happens w/ Omni Demo but not w/ Sempra

**Comment by christopherdude** [comment]
I'm not seeing this any more.  Perhaps my other fix fixed this one?

**Comment by davroth** [comment]
Still repro's w/ both FireFox and IE (haven't checked Chrome)

**Comment by patrickomni** [comment]
While I thought this was a resource not included, I think the culprit was the following:

```
        //var plot = $.plot($("#area-chart"), [{ label: '<% = R(YAxisResourceName, true)%>', data: items }], options);
```

Even though this line of JavaScript is commented out, the <% %> ASP still fires.  This YAxisResourceName did not have a valid value because it wasn't expected to fire at all; and since this would only be for the graph, that is why tank and fridge did not work.

```
//var plot = $.plot($("#area-chart"), [{ label: '<% // = R(YAxisResourceName, true)%>', data: items }], options);
```

So I added the extra commenting after <% ("//") and propped.

I'm still perplexed as to why it is only doing this as Dave; however, I did login as Dave to try to repro after the fix and cannot.
